Client/Frontend/Browser/PageZoom/PageZoomView.swift

```diff
@@ -0,0 +1,208 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveUI
+import WebKit
+import Data
+import Shared
+import BraveShared
+
+private struct ZoomView: View {
+  private let color = Color(UIColor.braveLabel)
```

Comment:
Just use `Color(.braveLabel)` in each place individually, setting it like this without any doc or proper naming is confusing